app/templates/index.html

```diff
@@ -29,6 +29,19 @@
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
 
+  {% if current_user.has_role('admin-manager') %}
```

Comment:
Bit of inconsistent indentation in templates? I'm using 2 spaces for everything in them :man_shrugging:

Reply:
I'll take a look in a bit and fix up.

Reply:
Need to figure out how to get PyCharm to do this.

Reply:
Yeah
The `column selection mode` in the edit tab is pretty useful for this i find but ot found a way to get it to do automatic indenting by 2 in templates

Reply:
Looks like it might be a paid-for feature.

Reply:
Think I've fixed it.